src/runtime/components/forms/SelectMenu.vue

```diff
@@ -413,8 +413,18 @@ export default defineComponent({
       }
 
       return (props.options as any[]).filter((option: any) => {
-        return (props.searchAttributes?.length ? props.searchAttributes : [props.optionAttribute]).some((searchAttribute: any) => {
-          return ['string', 'number'].includes(typeof option) ? option.toString().search(new RegExp(query.value, 'i')) !== -1 : (option[searchAttribute] && option[searchAttribute].search(new RegExp(query.value, 'i')) !== -1)
+        const searchable = typeof props.searchAttributes === 'string'
+          ? props.searchAttributes.split(',').map(attribute => attribute.trim())
```

Comment:
Still don't understand why you need to split `searchAttributes` as it's supposed to be an array of string. Also, when `searchAttributes` is not filled, it's supposed to fallback on searching on `props.optionAttribute`, why remove this behaviour?

Reply:
Another brain fart. I'll check it out later.

Sorry for so much trouble reviewing this.

Reply:
Fixed. Also, it uses an array of property names only.